packages/app/package.json

```diff
@@ -39,7 +39,13 @@
   },
   "peerDependencies": {
     "react": ">=16",
-    "react-dom": ">=16"
+    "react-dom": ">=16",
+    "react-error-boundary": "3.1.4"
+  },
+  "peerDependenciesMeta": {
+    "react-error-boundary": {
+      "optional": true
```

Comment:
Is `react-error-boundary` optional given the re-throw of errors is done in `onError` of `ErrorBoundary` ?

Reply:
It's optional for the consumer, not for the codebase, of that makes sense.

Reply:
Yeah, no, this doesn't work at all. I have a feeling that the combination [ `peerDependencies` + `{ optional: true }` ] is similar to putting `react-error-boundary` under `optionalDependencies`, in the sense that `@h5web/app` should work regardless of whether `react-error-boundary` is installed or not, which is not what I was after here.

FYI, the difference with `optionalDependencies` is that the package manager tries to install `optionalDependencies` but does not error out if their installation fails. Whereas the package manager does not try to install `peerDependencies` at all.

I'll revert this change.